ggshield/verticals/secret/repo.py

```diff
@@ -62,14 +63,14 @@ def scan_commits_content(
     matches_ignore: Iterable[IgnoredMatch],
     scan_context: ScanContext,
     progress_callback: Callable[..., None],
+    commit_scanned_callbacks: Callable[[Commit], None],
```

Comment:
nit: should be `commit_scanned_callback` (no ending `s`) because it's a single callback (called multiple times).